verilog/blink/Makefile

```diff
@@ -40,7 +40,7 @@ blink.json: blink.v
 
 # Use **nextpnr** to generate the FPGA configuration.
 # This is called the **place** and **route** step.
-blink.asc: blink.json ../pcf/fomu-hacker.pcf
+blink.asc: blink.json
```

Comment:
This should still probably depend on the pcf file?

Reply:
The point is that the PCF is different, depending on `FOMU_REV`. However, the PCF is not specified as a separated variable, but included in PNRFLAGS. Therefore, I think that the PCF file should be defined in a different variable, and that would be used as a dependency.

I didn't do these modifications in this PR because I wanted to "just convert the existing Verilog example". This enhancement would apply to both of them.